src/main/java/seedu/duke/logic/parser/Parser.java

```diff
@@ -121,7 +121,7 @@ public boolean extractType() throws CustomException {
 
         com.executeCommand();
 
-        if (!command.equals("/addfav") && !command.equals("/execfav")) {
+        if (!command.equals("/addfav") && !command.equals("/execfav") && com.isValid()) {
```

Comment:
```suggestion
        if (com.isValid()) {
```

First 2 conditions will be redundant as not will not be valid